modules/drivers/mongo/src/metabase/driver/mongo/execute.clj

```diff
@@ -176,36 +178,19 @@
                []
                (reducible-rows context cursor first-row (post-process-row row-col-names))))))
 
-
-(defn- connection->database
-  ^MongoDatabase
-  [^DB connection]
-  (let [db-name (.getName connection)]
-    (.. connection getMongoClient (getDatabase db-name))))
-
-(defn- start-session!
-  ^ClientSession
-  [^DB connection]
-  (.. connection getMongoClient startSession))
-
-(defn- kill-session!
-  [^MongoDatabase db
-   ^ClientSession session]
-  (let [session-id (.. session getServerSession getIdentifier)
-        kill-cmd (BasicDBObject. "killSessions" [session-id])]
-    (.runCommand db kill-cmd)))
-
 (defn execute-reducible-query
   "Process and run a native MongoDB query."
```

Comment:
I suggest documenting when things expect `mongo.connection/*mongo-client*` to be set.

Reply:
Done.